srcpkgs/kvmtool/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'kvmtool'
+pkgname=kvmtool
+version=3.18.0
+revision=1
+_githash=41d773e2070aabd3c4ff20c059f45c3e9a0f9d20
+wrksrc="${_githash}"
+build_style=gnu-makefile
+make_use_env=no
```

Comment:
Any string != "" will **enable** `make_use_env`. `=no` is very missleading as it actually *enables* it.

Reply:
Indeed, thanks. Pushed a fix. I noticed that quite a few templates use `make_use_env=compliant`, which may also be confusing (this isn't documented and as you properly point out makes no difference). Other templates do `make_use_env=1` which looks better, we might though like to explicitly mention this in the Manual in order to make things more consistent?